ui/app/rif/components/InputWithSubmit/index.js

```diff
@@ -0,0 +1,19 @@
+import React, {useState} from 'react';
+
+const InputWithSubmit = (submit) => {
```

Comment:
Please refactor this to be like the standar that we are using, we use classes and extend from component, we try to avoid functional components in this project.

Reply:
I agree with this, just because the hole project is using classes, to keep it standard